src/Transition/CreateNewIDFUsingRulesV8_9_0.f90

```diff
@@ -810,6 +810,21 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                    END IF
                  END IF
 
+             CASE('OUTPUT:TABLE:SUMMARYREPORTS')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 CurField = 0
+                 DO TempArgsNum=1,CurArgs,1
+                   ! Skip blank fields
+                   IF (.NOT. InArgs(TempArgsNum) == Blank) THEN
+                     CurField = CurField + 1
+                     OutArgs(CurField) = InArgs(TempArgsNum)
+                   END IF
+                 END DO
+                 IF (CurField < CurArgs) THEN
+                   Nodiff=.false.
+                   CurArgs = CurField
+                 END IF
```

Comment:
OK, so just loop over the entire set of old fields, and if any blanks are there, skip them and mark the file as dirty.  No problem with this.